receiver/hostmetricsreceiver/internal/scraper/processscraper/ucal/cpu_utilization_calculator.go

```diff
@@ -4,21 +4,13 @@
 package ucal // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/processscraper/ucal"
 
 import (
+	"fmt"
 	"time"
 
 	"github.com/shirou/gopsutil/v4/cpu"
-	"go.opentelemetry.io/collector/featuregate"
 	"go.opentelemetry.io/collector/pdata/pcommon"
 )
 
-var normalizeProcessCPUUtilizationFeatureGate = featuregate.GlobalRegistry().MustRegister(
-	"receiver.hostmetrics.normalizeProcessCPUUtilization",
-	featuregate.StageBeta,
```

Comment:
Typically we move this to stable first. You don't need to change much, just do `var _ =` at the top and switch to `featuregate.StageStable`